### DISCUSSION TITLE
Modify background SV by an offset in follow mode

### DISCUSSION THREAD

**Discussion by billysdomain** [artisan-roaster-scope/artisan#1444] [Q&A]
Hi,

Is it possible to have the Artisan PID SV update in background follow mode as {Background BT + Offset} rather than just {Background BT} where the offset may be +20 for example?

I know I can move the background around, but I want to keep that aligned and just change the SV

Many thanks

**Comment by MAKOMO** [comment]
No, but you can just move your background down by 20 degrees in the background dialog (menu >> Roast >> Background)

**Comment by billysdomain** [comment]
![roastoffset](https://github.com/artisan-roaster-scope/artisan/assets/66838638/95e39c8c-5d2d-49ee-8756-def6e7dae735)

That is what I have been doing; my system is most stable when its PID settings (actually mostly just P) are tracking under the setpoint, as the response from going over the SP is so slow to recover.

Moving the background moves all the data points and doesn't look the simplest on screen, so I was looking for a way to keep the background aligned but still have the offset processed.

Do you have any other ideas on how I could do this?

**Comment by hermetheuscoffee** [comment]
Try using the PID "lookahead" value (which can also be changed via buttons). It's in the "Control" menu. Lookahead does what the name implies... it looks AHEAD for the SV by the number of seconds you specify for the value. So for instance, if you have a value of 10 specified, then the set value at the VERY start of your will come from the 0:10 mark on your background curve graph. It's a way to do exactly what you're looking for, while still keeping the visual display of your graph as you would like.
